### ISSUE TITLE
Submissions from old forms are failing

### ISSUE THREAD

**Issue by sadumah** [getodk/collect#376]
<!-- 

Forms that used to successfully indicate SUCCESSFUL are now indicating FAILED... Nothing has been changed from my end or on the Aggregate server. What could have happened?
Initially when this problem started, I could still find my data had been pulled despite the report that pulling had failed but now it is actually totally FAILING and no data is being pulled

**Comment by sadumah** [comment]
Forms that initially used to successfully pull data are currently failing to pull successfully.
It used to display failed but I could still find data had been pulled but currently it is totally failing to pull data from Aggregate server using briefcase v1.4.9
I haven't changed anything on the forms nor on the aggregate server

**Comment by yanokwa** [comment]
This does not seem to be about ODK Collect.

If there is an issue with ODK Briefcase, file it at https://github.com/opendatakit/briefcase/issues/new. Be sure to include sufficient detail so it can be reproduced or the issue will be closed.

**Comment by sadumah** [comment]
Hi there,
unfortunately its not the briefcase but ODK collect! New forms work
successfully but the old forms i created in Janury 2016 are the ones failing

**Comment by yanokwa** [comment]
If submissions from new forms are working, but submissions from old forms are not working, the problem is very likely server side.

**Comment by yanokwa** [comment]
Before we can help, we need answers to the following...

1. Software and hardware versions 
Collect v1.x.x, Android vx.x.x, device used, Aggregate v1.x.x,

2. Steps to reproduce the problem

3. Other information 
Things you tried to fix it, stack traces, related issues, suggestions on how to fix it...

**Comment by sadumah** [comment]
Submission is working but downloading from the server is what is an issue.
Infact the data that was there before is still downloading but new records
are failing to be pulled using briefcase. I can see the new records on
Aggregate and even download manually from the tablets using briefcase but I
can't download the new records from the Aggregate server using briefcase

**Comment by yanokwa** [comment]
It sounds like you have a problem with Briefcase and not Collect, so I'm closing this issue.

If you have a problem with Briefcase, open a new issue and file it at https://github.com/opendatakit/briefcase/issues/new. 

Provide the information in the template to help us replicate the problem or it will be closed.
